eventActions.js

```diff
@@ -538,7 +554,8 @@ ActionSetProp.prototype.toJS = function () {
     return {
         type: this.type,
         refToObjectProperty: this.refToObjectProperty.toJS(),
-        operand: this.operand.toJS()
+        operand: this.operand.toJS(),
+        refsToObjectProperty: this.refsToObjectProperty()
```

Comment:
this should be replaced by something similar like in lines 359-362

Reply:
done, partially I think that there is something missing in toJS.